core/dbt/task/clean.py

```diff
@@ -1,54 +1,54 @@
 import os.path
 import os
 import shutil
+from typing import List
 
 from dbt import deprecations
-from dbt.task.base import BaseTask, move_to_nearest_project_dir
 from dbt.events.functions import fire_event
 from dbt.events.types import (
     CheckCleanPath,
     ConfirmCleanPath,
     ProtectedCleanPath,
     FinishedCleanPaths,
 )
-from dbt.config import UnsetProfileConfig
+from dbt.task.base import (
+    BaseTask,
+    move_to_nearest_project_dir,
+)
 
 
 class CleanTask(BaseTask):
-    # Note: CleanTask is the last task that uses UnsetProfileConfig,
-    # and can be deleted once CleanTask no longer requires it.
-    ConfigType = UnsetProfileConfig
```

Comment:
Could we rip out `UnsetProfileConfig` at this point? 🔥 Or in a follow-up?

Reply:
I think in a followup would make the most sense!